lib/src/signed_video_h26x_auth.c

```diff
@@ -32,6 +32,8 @@
 #include "signed_video_internal.h"  // gop_info_t, gop_state_t, reset_gop_hash()
 #include "signed_video_tlv.h"  // tlv_find_tag()
 
+#include "signed_video_openssl_internal.h"
```

Comment:
Put together with the other includes, and in alphabetical order. Also, add comment on what is included like the others.